test/test_container.py

```diff
@@ -0,0 +1,27 @@
+import os
```

Comment:
Would you be willing to use Go for integration tests? Is there any reason to actually use Python?

I can tell you why I prefer Go:
- developers don't need to switch their language mindset when writing integration tests
- we don't need an extra linter and formatter
- sharing code between integration tests and the SUT
- pytest fixtures are cool... until they start being overwhelming. I think I have a preference for the lightweight nature of Go.

Reply:
Sure, I will give this a "go" (see what I did here ;)